M2/Macaulay2/packages/FastLinAlg.m2

```diff
@@ -855,6 +940,71 @@ projDim(Module) := opts -> (N1) -> (
     return i;
 );
 
+isGBDone := (myGB) -> (
+    --a temporary function for finding out if a gb computation is done.
+    myStr := status myGB;
+    if (0 < #select("status: done", myStr)) then return true else return false;
```

Comment:
@mikestillman -- is there a better way to do this?  If not, we should invent one.

Reply:
I agree.  I don't know a better way to do it though.